### ISSUE TITLE
I think Usage sample should use 4343 instead of 43

### ISSUE THREAD

**Issue by xembook** [nemtech/symbol-hd-wallets#34]
for example 

https://github.com/nemtech/symbol-hd-wallets/blob/main/README.md#generating-a-hd-wallet-catapult-mijin-and-mijintest-compatible

```
const childAccount = wallet.getChildAccount('m/44\'/43\'/0\'/0\'/0\'', NetworkType.MIJIN_TEST);
```

I think this is

```
const childAccount = wallet.getChildAccount('m/44\'/4343\'/0\'/0\'/0\'', NetworkType.MIJIN_TEST);

```

thanks!

**Comment by xembook** [comment]
how about this?

**Comment by xembook** [comment]
this pull request closed.

**Comment by rg911** [comment]
Yeah, we have another pr pending on this, also the desktop wallet will get them in another pr,
